tests/TestDatePriortyView.cpp

```diff
@@ -67,3 +67,44 @@ TEST_F(PriorityViewTest, AllTasksSortedByDate) {
                   result_set[i].lock()->getTask().getPriority()      );
     }
 }
+
+TEST_F(PriorityViewTest, DateConstraintIsWorkingProperly) {
+    DatePriorityView tw;
+    tw.addToView(node1);
+    tw.addToView(node2);
+    tw.addToView(node3);
+    tw.addToView(node4);
+    std::vector nodes = {node1, node2, node3, node4};
+    std::sort(nodes.begin(), nodes.end(),
+            [] (const auto& lhs, const auto& rhs) {
+                return lhs->getTask().getDate() < rhs->getTask().getDate();
+    });
```

Comment:
Why do you use sort ? I hope it's only for cheking data

Reply:
We need sort to make our tasks ordered by date for sure. If we does not the test of View does not make sense.

Reply:
*edit. We FIRST add them to View and THEN sort.

Reply:
got it